components/GlobalWarnings.js

```diff
@@ -25,8 +27,13 @@ const GlobalWarningContainer = styled.div`
  */
 const GlobalWarnings = ({ collective }) => {
   const { LoggedInUser } = useLoggedInUser();
+  const [hasFreezeModal, setHasFreezeModal] = React.useState(false);
 
   if (collective?.isFrozen) {
+    const isLoggedInUserHostAdmin = LoggedInUser?.memberOf?.some(
+      member => collective.host?.id === member?.collective?.id,
+    );
```

Comment:
If `collective.host` is null, then we can end up with `null === null` which is true. Let's make it simpler & safer:

```suggestion
    const isLoggedInUserHostAdmin = Boolean(LoggedInUser?.isHostAdmin(collective));
```